indico/modules/events/abstracts/templates/_abstract/review.html

```diff
@@ -0,0 +1,85 @@
+{% from 'events/abstracts/_abstract/_common.html' import render_edited_hint %}
+
+{% macro render_abstract_reviews(abstract) %}
+    <div id="abstract-reviews" class="i-timeline">
+        {% set reviews = abstract.reviews|selectattr('track', 'equalto', none)|list %}
```

Comment:
`selectattr('track', 'none')` should be enough (there's a `none` test in jinja, so no need to use `equalto` for it)